gsoc2018-bharat/src/create_db.py

```diff
@@ -0,0 +1,69 @@
+import json
+from gensim.models import FastText
+from gensim.models import KeyedVectors
+import torch
+from torch.autograd import Variable
+import numpy as np
+import Encoder
```

Comment:
I think `Encoder` has been removed / renamed, so maybe you need to reflect the change here